py/examples/device_tapdetector_example/README.md

```diff
@@ -0,0 +1,31 @@
+## Tap Detector
+### About
+The tap detector is designed to take input from two differnt sources of data and to "match" them together by detecting taps between the two devices. For example, two devices recording at the same time can be tapped against each other. Then, the resulting data files from both devices can be used with the tap detector to find the corresponding taps in either file so that their timestamps can be aligned. Data smoothing is done using a hann filter, whose window is specified by the user for each source of data.
+
+Sample data is included in a .zip file to use. The following command/parameters works well for the sample data provided. Note that the sample data includes noise at the end.
+```py .\py\emotibit\device_tapdetector.py -sof .\emotibit5_AX.csv .\emotibit5_AY.csv .\emotibit5_AZ.csv -sod 3 -sot LocalTimestamp -soa AX AY AZ -stf .\cyton5.txt -std 3 -stt " Timestamp" -sta " Accel Channel 0" " Accel Channel 1" " Accel Channel 2" -n1 EmotiBit -n2 Cyton -o taps5 -w2 13 -h2 0.08```
+
+### Command Line Usage
+The tap detector is designed to be used with the command line. The tap detector is designed to work whether all dimensions of your data (e.g. X, Y, Z) are in one file or are in multiple files. The following example that is shown uses Emotibit files which store dimensions seperately and a cyton file where all dimensions are in one file:
+
+```$py device_tapdetector.py -sof .\emotibit_AX.csv .\emotibit_AY.csv .\emotibit_AZ.csv -sod 3 -sot LocalTimestamp -soa AX AY AZ -stf .\cyton.csv -std 3 -stt " Timestamp" -sta " Accel Channel 0" " Accel Channel 1" " Accel Channel 2" -tw1 0 10 -n1 EmotiBit -n2 Cyton -w1 1 -w2 13 -h1 0.1 -h2 0.4```
```

Comment:
If we are asking people to run this specific command, i think we should also clarify what the file structure of the files should be.
Ex: where do you run this command from? Where are the data files relative to this path?

Reply:
Added where those should be.